<ui version="4.0" >
 <class>settings_Dialog</class>
 <widget class="QDialog" name="settings_Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EWItool - Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>271</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pushButton" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>280</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="libraryLocation_lineEdit" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Library Directory</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title" >
    <string>EWI Patch Exchange</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>User ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pxUserID_lineEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>30</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maxLength" >
     <number>32</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="pxPasswrd_lineEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>70</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pxServer_lineEdit" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>110</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="test_pushButton" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>30</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="pxTest_label" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>Connection is untested</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
